OpenRA.Mods.Common/Traits/Render/SelectionDecorationsBase.cs

```diff
@@ -93,8 +99,15 @@ IEnumerable<IRenderable> DrawDecorations(Actor self, WorldRenderer wr)
 				foreach (var r in RenderSelectionBars(self, wr, displayHealth, displayExtra))
 					yield return r;
 
-			if (selected && self.World.LocalPlayer != null && self.World.LocalPlayer.PlayerActor.Trait<DeveloperMode>().PathDebug)
-				yield return new TargetLineRenderable(ActivityTargetPath(self), Color.Green, 1, 2);
+			if (selected && self.World.LocalPlayer != null)
+			{
+				// World.LocalPlayer and Player.PlayerActor do not change.
```

Comment:
I find this comment unnecessary. A more important comment would be to mention why this can't be done in `NotifyCreated.Created`. In the suggestion I only assume that's how it works.

```suggestion
				// DeveloperMode is only initialised after INotifyCreated.Created is called
```

Reply:
It was new to me. But perhaps everyone else knows. It explains why it is save to cache.

The comment you suggest also does not add a lot. DeveloperMode is available only that LocalPlayer is just not set yet, which is in the if statement.

Lets me remove both?

Reply:
Sure